media/class-diagram.drawio

```diff
@@ -1 +1 @@
DRO8hnah2DG1MZr/51nRvcEiWDMa7FX6bllOqoXNNsmxkpdHF9KousqlHfPa6PNGYKOZuU1BBQqddXf1HGC3Kj3sKl5yhT/X7FLzLxsbZqJJD6DMb594+XWNK8xA1sHFDZ9k4Q1OEtUa2I5bvyNMa+Q8bUuZhGZvefWNzZdkayxxQw7amA1u7jh6COPKiV7U2a29txk5BGmetZHDWSmcVwedbLIzEpH9eb800ClZzYLLnp16Xj+aT1GueXclQr/mXLLDUHpB6LQlWnnrNg1WCel3KF/PhUjLtkGelnEqLTkXAq+gD9SpwdT7By7KksOVAE8LKx/tJGWHNOh6YCqgcz8mw8nLCGnU8UKgBQ1V5nuY9j81EAlSoKbNYgdTUnsaaTpMS6VBuO9pkuVd2sWPzSLEHdmWxXXU/4pRJ/70JOdERNsCYE722PFjdDzrJA6znUSfqCpisY8Vk2mMyWTCIbr/lpRwPk8noUJk9FyYjnKJYOoxbZjI2mzfjHslj2I6AbfeAxUDZueMsho6uIbIYTpZT91mMPMD6zmJ4m8SXt8HKV7pM22yGkehN1wX0IstyL+h3JbLseUQEdKgPb3Y7/nbxgAPb/wYi/OtStoLzLKw24V+Hgtvaew2xN90B+333QQFbEVie9N8ssAJJmkOP6FSH1RkJeoSaUDWg3ghQPedsSF10c4awXl5/OqSlHbmy1A1m5Ws1mw5pQL2Uhg8VV207hngkVT3/5BUqm+SsNpiNdxNhwmg+z0qNdyCMxhC3uK5QVQOqVIrRnAyrKFGtjdEIFGw4Z0ZDEes0o0FWkYlY7MYM4aC/yVAa1h/VTWmg+nbAHygy0ziZsWmIR+luhtLdRLxe/3Q3A8ozSneTAGzrupuhimvWAGvbuptZTXfrUzbHaayUQtlyNoepmVe0TBSdCSyTnqsqS+X0ZiPQW/fzOkwo4HU8r4M+qgHmdZhQtxIHq6V6KNLQ6nlSh3m+yX7CYZjSrMCW3YPF5KkabCRFuEgW05EpWKFCglOoWt7QZs5gWQ3UNzR7l0loDjeT0DylXlpLHqfbaYS61hh21Sri9t7jGH3zOIymabFCt3B6OatWjjrrcQwm8cGxzQY8DlRIu+5x7OF6HKh6dt7jSEOr7x4HCpsHRGoVOms+Zz1j/SoRyIKaraqNxg7m/kTMLChmAvSGHlipDmvb6T/0/ENex5RC2fI6BjE5yJbNkEXRdQzbkT0yix11P5xiQZW340sNOroGuNSwTlEi21lqyEOr5+EUq1pOaO+dgvCuFuUUuucUoBDbdadgDdcpnCIWtuQUpKHVd6cw8LeOWK6gk6DPpZWXADIVmtxjXwVho6JncAVrA1aOWTCC1mjUQJTcglLqJ2+jipmJjcpq46rC/MiQEVfjFDPjlu89+20INpRSH/3XnZD6xd/7bpSBiKmWDDG12RdN2FBMDTafV9s7SHcgPGAcKkRlv2OiPkT1w/Ro6PJ4dVg5+w4afXWIDSXHG7xcK6rQLlWw6ZI1owqCdoLo22vOmCtA4VVxBTige8QVoBqbi6P/SP5UoPaOLkAlLEEhQZSuqBWQEglCfUAKJAIPnfdVhxUJr/3rgnXgaqctvZx4HWonYmiZRrPPK8udDqM9sXmdkuROa9SC3GlDubOECKr1SxNZoy5juDqc7waaNupArTMg/CQOZoEfbXivP/ycfa1egFgc8P3JLnWUfFYDrK1nl2pQ5+aMVuWBmvdAps3kJiHE8UFZTG5Y8TZLg9PRylv6autCfkyfJqFxDKu+vQsaFEUDMghfBq+4HIEmVztrFk2Bch9DJwtHAMvV0njzf224DizvmBK6gyJLBmbLice6waxbkeZeua5r6cZII/9nsoeFq9Kyy+HybnuQlKz1LytZG3BassbJS+7+O90kQtb33GStmlzfe88hnIxMXUzbO1YcVqlny6gIb1lhe0LNvbW8qtqvs0terZHsZupN++R8RsN1PggK3p3fFCMPr757HiRQFuGcPI9wYLgjnsdGbMdkcXGk73GQu1c07Z73sQ32vq0G6oFZNMOyR96HjuFBep9Tiiq05H3k4SVrP3Zr3gfq2qUJoSq0Vn9ozRqNGB9Bq/LmhVWTE+kyjkvuYPZ86g4nsMY7XRuBNTQ0iVeULmVDuW2+ZLPr62NVXcdmVV2HiRx0iCuxd93MSh31TyZGA5aJUQ/LV8jDq/crdagR320zORRZao8smYghS7rLIUs6h72cfyashaCQqyqoguF8UhoSz7LqS1zRodQL8Bt84kp1YHkZSTxg6yuiCkXgb0/LB5Xi2uo63FWuZb/FQpU251qGnBCpixtad/yKSoisA9j2/QqUd8cYh763Uo6lPcfisCkoyrHkTLa0WsWgi1plo7lPnkWVN6gD2PY9C5QyAa7nXN8gW2VKDMzUUOHAZKf+rOHk99gZrDPaE42R5xkGXlIjI6AdNzmW3Zhsgq2wybFBNfDa39pNDgqoAzM5sw8mBwylFya3iPQn5ze+Q+HXcP5pYZh3j/eXUOC9SVZDwUqVl660agNUSnRU7F21uYwD5JaX5qXJIFpR8SxWbVyrhfJ1GGziL4MpG3nA2vaP9EorN5EcrGMYPvfy9izDFabHY8pbtDWKaalmP4BCoPIh5W15bxRSKNyffxlQ+TByy0U2iiOUyQZUz78GQA0OWWoUUBh5AeCdue5ZA6qWICuSIHtyL6+foiezJD9OHyhbrQoWeW1EC3DNFBNaIZVNGj9WChgZTrEjSSVdbZc5j+TMcS5w/Sh30L7h7rYg7NUE6tpDYbI7L9mdPaJGnCmYe/eudmgPBTMQpO+h4FpC12odCOfZC3jTUanxdnn/BF9lguIonLYa3T7RHFiSNpqiwgirZfMEH7p+VDlo3+GkcZSmPY4rbdce6Ek3naQWW/ZfZ/fwuaz/QWmVpFr9T+fKHcic01B/d/Dx76dzxQ4aREuSC7KluyDyMcI4zh8eeevFVzz1kyP+Dw==</diagram></mxfile>
```

Comment:
There's a visual problem with the cardinality `1` next to the `StmtLstExprAst`.

Reply:
Good catch. Fixed it